src/components/AccessDenied/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+
+function AccessDenied() {
+  return (
+    <div>
+      <h1>Access is Denied</h1>
+      <p>Please Log in to view the pages of your visits</p>
```

Comment:
We should add a link to the login page here. Also we should remove the mention of Visits. This page will be  displayed when the user tries to access any unauthorized route, not just the Visits